tests/unit/data/test_extract_metadata.py

```diff
@@ -35,16 +35,16 @@ def cube_1():
     @staticmethod
     @pytest.fixture
     def cube_2():
-        latitude = DimCoord(np.linspace(1, 100, 100),
+        latitude = DimCoord(np.linspace(1, 100, 200),
```

Comment:
I don't know whether it matters, but latitude is usually between -90 and +90 (or 0 and 180 sometimes if you need to avoid negative numbers by offsetting everything by +90).

It'd be good to use realistic values where possible

Reply:
I went with this originally as `cube 2` dimensions are in meters rather than degrees, just in order to test different kinds of cubes. But then you're right about `cubes 3` and `4`, and we just end up comparing the raw values anyway, which... is that wrong?? Do we need to add some kind of check?

Reply:
ah yea, CRS should handle that. I guess that if `cube 2` is using metres, then it might be using a UK Ordnance Survey grid? Would be worth changing the variable names if they don't actually refer to lat/lon in this fixture

Reply:
I think it's just a [generic x-y](http://vocab.nerc.ac.uk/standard_name/projection_x_coordinate/) mapping; I'll change lat/lon to x/y. Should we create a ticket for the CRS stuff?

Reply:
> Should we create a ticket for the CRS stuff?

Yea, might be a good idea to separate that. I had a quick look and it wouldn't have worked before anyway, because the type of the CRS object returned isn't right for `SpatialExtent` (which expects an instance of `edr_server.core.models.crs.CrsObject`), so `_cube_to_polygon` needs to do a conversion